src/main/flight/mixer.c

```diff
@@ -512,6 +512,8 @@ void calculateThrottleAndCurrentMotorEndpoints(void)
         if((rcCommand[THROTTLE] <= (rxConfig()->midrc - flight3DConfig()->deadband3d_throttle))) {
             motorOutputMax = deadbandMotor3dLow;
             motorOutputMin = motorOutputLow;
+            throttlePrevious = rcCommand[THROTTLE];					//3D Mode Throttle Fix #3696
+            throttle = rcCommand[THROTTLE] - rxConfig()->mincheck; //3D Mode Throttle Fix #3696
```

Comment:
Yeah, to make it match the 'positive throttle' case below, this should be `rcCommand[THROTTLE] - (rxConfig()->midrc + flight3DConfig()->deadband3d_throttle)`.

(And the case below should be `rcCommand[THROTTLE] - (rxConfig()->midrc - flight3DConfig()->deadband3d_throttle)` IMO.